doc/ikfast/ikfast_tutorial.rst

```diff
@@ -73,210 +53,79 @@ Inside your catkin workspace's ``./src`` directory: ::
-
-You should consult the OpenRAVE mailing list and ROS Answers for information about 5 and 7 DOF manipulators.
-
-Create Plugin
--------------
-
-Create the package that will contain the IK plugin. We recommend you name the package ``"$MYROBOT_NAME"_ikfast_"$PLANNING_GROUP"_plugin``: ::
-
- cd ~/catkin_ws/src
- export MOVEIT_IK_PLUGIN_PKG="$MYROBOT_NAME"_ikfast_"$PLANNING_GROUP"_plugin
- catkin_create_pkg "$MOVEIT_IK_PLUGIN_PKG"
-
-Build your workspace so the new package is detected (can be 'roscd'): ::
-
- catkin build
-
-Create the plugin source code. Run ``rosrun moveit_kinematics create_ikfast_moveit_plugin.py -h`` to see optional arguements: ::
-
- rosrun moveit_kinematics create_ikfast_moveit_plugin.py "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
-
-For the Panda we need to use some of the optional parameters: ::
-
- rosrun moveit_kinematics create_ikfast_moveit_plugin.py --moveit_config_pkg="panda_moveit_config" --robot_name_in_srdf="panda" "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
-
-
-Or without ROS: ::
-
- python /path/to/create_ikfast_moveit_plugin.py "$MYROBOT_NAME" "$PLANNING_GROUP" "$MOVEIT_IK_PLUGIN_PKG" "$BASE_LINK_NAME" "$EEF_LINK_NAME" "$IKFAST_OUTPUT_PATH"
+  catkin build
 
 Usage
 -----
-The IKFast plugin should function identically to the default KDL IK Solver, but with greatly increased performance. The MoveIt configuration file is automatically edited by the moveit_ikfast script but you can switch between the KDL and IKFast solvers using the *kinematics_solver* parameter in the robot's kinematics.yaml file: ::
+The IKFast plugin can be used as a drop-in replacement for the default KDL IK Solver, but with greatly increased performance. The MoveIt configuration file should be automatically edited by the generator script but in some cases this might fail. In this situation you can switch between the KDL and IKFast solvers using the *kinematics_solver* parameter in the robot's kinematics.yaml file: ::
 
- rosed "$MYROBOT_NAME"_moveit_config kinematics.yaml
+  rosed "$MYROBOT_NAME"_moveit_config kinematics.yaml
 
 Edit these parts: ::
 
  <planning_group>:
-   kinematics_solver: <myrobot_name>_<planning_group>_kinematics/IKFastKinematicsPlugin
- -INSTEAD OF-
-   kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
+   kinematics_solver: <myrobot_name>_<planning_group>_ikfast_plugin/IKFastKinematicsPlugin
 
 Test the Plugin
 ^^^^^^^^^^^^^^^
-
 Use the MoveIt RViz Motion Planning Plugin and use the interactive markers to see if correct IK Solutions are found.
```

Comment:
I think it would be really nice to add more information on how to actually test this with RViz.

Reply:
Sorry, but using rviz interactive markers was handled in another tutorial already.